src/views/TwabRewards/SummaryWell.tsx

```diff
@@ -0,0 +1,18 @@
+import classNames from 'classnames'
+
+export const SummaryWell = (props) => {
```

Comment:
We could probably share a lot of the summary/final confirmation modal code across both tools